cmd/get/clusters/runner.go

```diff
@@ -67,14 +68,21 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	var resource runtime.Object
 	{
 		options := cluster.GetOptions{
-			Provider: r.provider,
+			Provider:  r.provider,
+			Namespace: metav1.NamespaceDefault,
```

Comment:
is this necessary if we are going to overwrite it with either `NamespaceAll` or whatever is in the config file?

Reply:
That was there in case the type cast failed... But I just realized that we don't actually need that type cast anymore, so we don't need the default either